files/image_config/platform/rc.local

```diff
@@ -35,8 +35,18 @@ if [ -f /host/image-$sonic_version/platform/firsttime ]; then
         mv -f /host/old_config/* /etc/sonic/
     elif [ -f /host/minigraph.xml ]; then
         mv /host/minigraph.xml /etc/sonic/
+        if [ -f /etc/sonic/init_cfg.json ]; then
```

Comment:
>if [ -f /etc/sonic/init_cfg.json ]; then [](start = 8, length = 40)

add some comment for TODO. For example, currently we still load minigraph using -m, we need to remove it in the future.